Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -56,116 +62,97 @@ public void AfterEachTest()
 
-            string actualIncome = browser.FindElement(By.XPath("//input [@id = 'income']")).GetAttribute("value");
-            
+            CalculatorPage calculatorPage = new CalculatorPage(browser);
+            string actualIncome = calculatorPage.ActualIncome.GetAttribute("value");
             Assert.AreEqual("110.00", actualIncome);
-            string actualInterest = browser.FindElement(By.XPath("//input [@id = 'interest']")).GetAttribute("value");
+
+            string actualInterest = calculatorPage.ActualInterest.GetAttribute("value");
             Assert.AreEqual("10.00", actualInterest);
         }
 
         [Test]
         public void PositiveTestDepositAmountIsMandatoryField()
         {
-            browser.FindElement(By.XPath("//td[2]//input[@id = 'amount']")).SendKeys(" ");
-            browser.FindElement(By.XPath("//input [@id = 'percent']")).SendKeys("10");
-            browser.FindElement(By.XPath("//input [@id = 'term']")).SendKeys("365");
-            browser.FindElement(By.XPath("//input[@type][2]")).Click();
-
             CalculatorPage calculatorPage = new CalculatorPage(browser);
-            calculatorPage.CalculateBtn.GetAttribute("value");
 
-            Assert.AreEqual("CalculateBtn", calculatorPage.CalculateBtn);
+            new CalculatorPage(browser)
+                .EnterCalculatorData365days(" ", "10", "365");
 
-            /*string actualIncome = browser.FindElement(By.XPath("//input [@id = 'income']")).GetAttribute("value");
-
-            Assert.AreEqual("0.00", actualIncome);
-            string actualInterest = browser.FindElement(By.XPath("//input [@id = 'interest']")).GetAttribute("value");
-            Assert.AreEqual("0.00", actualInterest);*/
+            Assert.AreEqual(false, calculatorPage.CalculateBtn.Enabled);
         }
 
         [Test]
         public void PositiveTestRateOfInterestIsMandatoryField()
         {
-            browser.FindElement(By.XPath("//td[2]//input[@id = 'amount']")).SendKeys("100");
-            browser.FindElement(By.XPath("//input [@id = 'percent']")).SendKeys(" ");
-            browser.FindElement(By.XPath("//input [@id = 'term']")).SendKeys("365");
-            browser.FindElement(By.XPath("//input[@type][2]")).Click();
-            string actualIncome = browser.FindElement(By.XPath("//input [@id = 'income']")).GetAttribute("value");
+            CalculatorPage calculatorPage = new CalculatorPage(browser);
 
-            browser.FindElement(By.XPath("//button [@id = 'calculateBtn']")).Click();
+            new CalculatorPage(browser)
+                .EnterCalculatorData365days("100", " ", "365");
 
-            Thread.Sleep(1000);
-            Assert.AreEqual("0.00", actualIncome);
-            string actualInterest = browser.FindElement(By.XPath("//input [@id = 'interest']")).GetAttribute("value");
-            Assert.AreEqual("0.00", actualInterest);
+            Assert.AreEqual(false, calculatorPage.CalculateBtn.Enabled);
```

Comment:
For boolean checks you should use 'IsTrue' or 'IsFalse'

```suggestion
            Assert.IsFalse(calculatorPage.CalculateBtn.Enabled);
```

Reply:
done, thanks